internal/config/config.go

```diff
@@ -434,3 +440,17 @@ func ValidateVisibility(visibility string) bool {
 
 	return false
 }
+
+// ValidateRerun checks the retries, maxAttempt and passThreshold settings
+func ValidateRerun(rerun Rerun) bool {
```

Comment:
Should this be compared with `sauce.retries` too ? At least provide a warning ?

Reply:
added